contrib/ansible/roles/download/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+- name: Delete k3s if already present
+  file:
+    path: /usr/local/bin/k3s
+    state: absent
+
+- name: Download k3s binary x64
+  get_url:                                                           
+      url: https://github.com/rancher/k3s/releases/download/v0.2.0/k3s
```

Comment:
May be create a variable for release, so it can be overriden while running playbook.

Reply:
Done in [9f35048](https://github.com/rancher/k3s/pull/217/commits/9f35048e7fd2eef7805237e6007da4db7d8eca13)

Reply:
Thanks for quick addition! :+1: